### PR TITLE
[CI] Add some codeowner

### PR DESCRIPTION
**Pull request by SbloodyS**
## Purpose of the pull request

add some codeowner

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -24,6 +25,10 @@
 /dolphinscheduler-common/ @caishunfeng
 /dolphinscheduler-standalone-server/ @kezhenxu94 @caishunfeng
 /dolphinscheduler-log-server/ @caishunfeng
+/dolphinscheduler-datasource-plugin/ @caishunfeng
+/dolphinscheduler-dist/ @kezhenxu94 @caishunfeng
+/dolphinscheduler-meter/ @caishunfeng
```
Comment (kezhenxu94) [inline .github/CODEOWNERS]:
```suggestion
/dolphinscheduler-meter/ @kezhenxu94 @caishunfeng
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -34,8 +39,11 @@
 /dolphinscheduler-tools/ @caishunfeng @SbloodyS @zhongjiajie
 /script/ @caishunfeng @SbloodyS @zhongjiajie
 /dolphinscheduler-python/ @zhongjiajie
-/dolphinscheduler-ui/ @songjianet
+/dolphinscheduler-ui/ @songjianet @Amy0104
 /docs/ @zhongjiajie @Tianqi-Dotes
+/licenses/ @kezhenxu94 @zhongjiajie
+/images/ @zhongjiajie @Tianqi-Dotes
+/style/ @kezhenxu94 @caishunfeng
```
Comment (kezhenxu94) [inline .github/CODEOWNERS]:
```suggestion
/style/ @caishunfeng
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (SbloodyS) [reply .github/CODEOWNERS]: Done.

File: .github/CODEOWNERS (hunk 2 above)
Comment (SbloodyS) [reply .github/CODEOWNERS]: Done.

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**